examples/javascript/simple_transfer.js

```diff
@@ -32,7 +32,8 @@ const example = async () => {
   console.log("This example will create two accounts (Alice and Bob), fund them, and transfer between them.");
 
   // Setup the client
-  const sdk = new aptos.Aptos();
+  const config = new aptos.AptosConfig({ network: aptos.Network.DEVNET });
```

Comment:
I don't think this is taking in the obj as param anymore? And default is devnet already?

Reply:
It does and always is. This is a configuration object.
Yes devnet is default but i think it helps to show the usage of the config option

Reply:
ohh I see, so its taking in `AptosSettings` as an object.